src/lib/planviz/abstract_visualizer.hpp

```diff
@@ -39,6 +39,7 @@ struct VizEdgeInfo {
   std::string color = "white";
   std::string font_color = "white";
   double pen_width = 1.0;
+  std::string style;
```

Comment:
This not having a default makes it default to solid line? Maybe add the default? 

After this is solved, mergytime ;)

Reply:
problem is: I have no idea what the default string is ;)